repo/packages/R/riak/1/command.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
@tamarrow can you help here? Do we have simple CLI in Python that compiles to a binary bundle?